lib/global_id/global_id.rb

```diff
@@ -10,12 +10,11 @@ class << self
     attr_reader :app
 
     def create(model, options = {})
-      if app = options.fetch(:app) { GlobalID.app }
-        new URI::GID.create(app, model), options
-      else
-        raise ArgumentError, 'An app is required to create a GlobalID. ' \
-          'Pass the :app option or set the default GlobalID.app.'
-      end
+      new URI::GID.create(get_app(options), model), options
+    end
+
+    def create_from(class_name, id, options = {})
```

Comment:
I really dislike this method name. Is there a better way we can communicate we're assembling from the bare parts here? Would `assemble` work?